project_script/main.py

```diff
@@ -29,49 +29,81 @@ def ask_file_name():
     return input("Insira o nome do arquivo para salvar \n")
 
 
-def ask_x_axys():
+def ask_x_axis():
     return input('Insira o cabeçalho do eixo x a ser considerado para o plote \n')
 
 
-def ask_y_axys():
+def ask_y_axis():
     return input('Insira o cabeçalho do eixo y a ser considerado para o plote \n')
 
 
-path_dark = ask_file_path("dark-CELIV")
+def is_bigger_data_frame(data_frame_1, data_frame_2):
+    return len(data_frame_1) < len(data_frame_2)
```

Comment:
Normally i see the first arg in the functions, are the arg that you want to made a comparison, a diff, or something, then, your function is called `is_bigger_data_frame`, and this name gives me the feeling that i'll have if the arg A it's bigger than arg B, but in this case, your fn return if the arg A it's lower than arg B. Maybe your statement could be `return len(data_frame_1) > len(data_frame_2)`. If you agree, and change, then check the integrity of the result of your whole script.

Reply:
I believe it'll break the code. I'd have to change all the parameters sense in other parts of the code to keep it running. In this case, I believe that the most simple is just to trade bigger for shorter. Working on.

Reply:
Or, maybe you could invert the algorithm. Is it right?

![image](https://user-images.githubusercontent.com/11571434/155774006-e8a273ca-6752-47b8-b347-b5a84aaa9fe2.png)

Reply:
Yeah, this also works.

Consider it done.